shotgun_api3/shotgun.py

```diff
@@ -270,6 +270,7 @@ def __init__(self):
         self.session_token = None
         self.authorization = None
         self.no_ssl_validation = False
+        self.multi_entity_update_modes = ['set','add','remove']
```

Comment:
I'd leave out the client side validation.  These will be checked server side and if the set of update modes ever changes on the server there will be an issue sync'ing up the python code.